build.gradle

```diff
@@ -36,6 +36,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
+        jcenter()
```

Comment:
I routinely get resolution errors locally and adding jcenter() resolves them. I see that jcenter() is added as a repository elsewhere in the project, so I don't see any issue in adding another option for resolving dependencies.